pinot-controller/src/main/java/org/apache/pinot/controller/util/TableSizeReader.java

```diff
@@ -78,8 +82,8 @@ public TableSizeDetails getTableSizeDetails(@Nonnull String tableName, @Nonnegat
     Preconditions.checkNotNull(tableName, "Table name should not be null");
     Preconditions.checkArgument(timeoutMsec > 0, "Timeout value must be greater than 0");
 
-    boolean hasRealtimeTable = false;
-    boolean hasOfflineTable = false;
+    boolean hasRealtimeTable;
```

Comment:
Since we need to read table config, directly read the table config to identify whether the table exists